test/img.spec.js

```diff
@@ -0,0 +1,58 @@
+// @flow
```

Comment:
is this a helper test that picturebook could provide as a default?

Reply:
it's fairly project-specific. I don't think it's really needed either, it's just a glorified version of

```
 cat ./picturebook-results.json | jq '.status' | grep "SUCCESS" > /dev/null && echo "success" || echo "failed" && exit 1
```

Reply:
lol, well that is a pretty specific command 😅 would be nice if there was a simpler way to do that.

Reply:
something like `picturebook check --output ./picturebook-results.json`?

Reply:
I do like that better! would we be able to kill the jest image test in that case? or does it still provide value above anything `picturebook check` would do?

Reply:
this gives us more info. It's up to us how much time do we want to invest on building a CLI tool for picturebook.

I can see some value to also use it as an interactive snapshot update and to generate the initial set of browser validations. Beyond that not sure what other use cases it could have

Reply:
👍